Source/EasyNetQ.DI.Tests/MockBuilder.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Collections.Generic;
+using RabbitMQ.Client;
+using RabbitMQ.Client.Framing;
+using Rhino.Mocks;
+using NUnit.Framework;
+
+namespace EasyNetQ.Tests.Mocking
+{
+    public class MockBuilder
+    {
+        readonly IConnectionFactory connectionFactory = MockRepository.GenerateStub<IConnectionFactory>();
```

Comment:
All these readonly fields can be changed to [auto-properties](https://www.simple-talk.com/dotnet/.net-framework/whats-new-in-c-6/)

```csharp
public List<IModel> Channels { get; } = new List<IModel>();
```